lists2safebrowsing.py

```diff
@@ -91,11 +167,20 @@ def find_hosts(disconnect_json, allow_list, chunk, output_file, log_file,
   categories = disconnect_json["categories"]
 
   for c in categories:
+    in_content_category = c.find("Content") != -1
+    if content_category == 'ONLY' and not in_content_category:
+      # If we're ONLY including the content category and this is NOT it,
+      # we can continue to the next category
+      continue
     # Skip content and Legacy categories as necessary
     if c.find("Legacy") != -1:
       continue
```

Comment:
If you wanted, you could add a comment here that says "Disconnect told us to ignore these."